src/content/reference/manifest.mdx

```diff
@@ -70,6 +70,8 @@ In order to refer to these images from anywhere in your [Okteto Manifest](/docs/
 
 You can build all these images by running `okteto build`, or `okteto build xxx` to build a single one.
 
+> if a `<service>` name includes the symbol `-`, the corresponding `OKTETO_BUILD_<service>` environment variables substitude the symbol `-` by `_`.
```

Comment:
Can you add an example?

Reply:
It's not something that's easy to understand just from the text. If it doesn't fit as a note we can make it a subsection or something like it.